argilla-server/pyproject.toml

```diff
@@ -56,6 +56,7 @@ dependencies = [
     # for CLI
     "typer >= 0.6.0, < 0.10.0", # spaCy only supports typer<0.10.0
     "packaging>=23.2",
+    "psycopg2-binary>=2.9.9",
```

Comment:
I think we might set the version for "psycopg2-binary>=2.9.9" a bit more flexible instead of the latest release. normally the server should used stand-alone but having some flexibility might be nice.

Reply:
In fact, I don't think this change should be included.